servermanagement/startServers_test.go

```diff
@@ -0,0 +1,64 @@
+package servermanagement
+
+import (
+	"testing"
+	"encoding/json"
+	"github.com/mozilla/mig/modules/netstat"
+	"github.com/SUMUKHA-PK/Raft-Distributed-Consensus/types"
+	"os"
+	"log"
+)
+
+func TestStartServer(t *testing.T){
+
+	file, err := os.Open("../server.config.json")
```

Comment:
This is the file opening error

Reply:
These errors are handled in the test for main, I was just reading it so that I can check if the IP address is not being used and to make a test configuration.

Reply:
If you wont use the variable use `file,_ :=` instead.